tests/conftest.py

```diff
@@ -100,3 +100,18 @@ def dummy_config(test_seed, dummy_split, dummy_dataset_name, dummy_model_name):
         "n_samples": 50,
         "random_state": test_seed,
     }
+
+
+@pytest.fixture
+def dummy_validation_scores(rng, test_n_samples):
+    return np.sort(rng.uniform(0, 1, test_n_samples)) * 100
```

Comment:
Very minor but is the sort needed here?

Reply:
Possibly not, but wanted to ensure sufficient correlation distance from the random scores in the tests.

Reply:
Add a comment so we remember why this is here if keeping it

Reply:
Done